app/routes/session.py

```diff
@@ -99,10 +98,10 @@ def login():
 
 
 def validate_jti(decrypted_token):
-    expires_at = datetime.utcfromtimestamp(decrypted_token["exp"]).replace(
-        tzinfo=tzutc()
-    )
-    jwt_expired = expires_at < datetime.now(tz=tzutc())
+    expires_at = datetime.fromtimestamp(
+        decrypted_token["exp"], tz=timezone.utc
+    ).replace(tzinfo=timezone.utc)
```

Comment:
Again, shouldn't need to replace when you pass in `tz` already.

Reply:
Removed, thanks!